locations/spiders/regal_nails_us.py

```diff
@@ -0,0 +1,15 @@
+from locations.storefinders.wp_store_locator import WPStoreLocatorSpider
+
+
+class RegalNailsUSSpider(WPStoreLocatorSpider):
```

Comment:
You'll need to add `time_format = "%I:%M %p"` as there is one location, yes just one location, that has hours specified in `9:00 AM` format.

Reply:
Added, my dev environment is MIA so have not re-run/which particular store is it to check?